README.md

```diff
@@ -137,7 +137,7 @@ Once ap-tools has started and the [tilt console](http://localhost:10350) is all
 
 ### CAS1 & CAS 3 (Delius)
 
-We login to CAS1 and CAS3 Systems using delius credentials. Anything can be used for passwords as we mock community API authentication using wiremock, and do not check passwords
+We login to CAS1 and CAS3 Systems using delius credentials. Anything can be used for passwords as we do not check passwords
```

Comment:
We technically still do mock the community-api authentication endpoint (yes, it's confusing!) 

How about changing it to this:

> We login to CAS1 and CAS3 Systems using delius credentials. Anything can be used for passwords as we mock the authentication used by hmpps-auth using wiremock, and do not check passwords